Intersect.Server/Core/Commands/SetVarCommand.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Intersect.Server.Core.CommandParsing;
+using Intersect.Server.Core.CommandParsing.Arguments;
+using Intersect.Server.Localization;
+using Intersect.Server.Database;
+using Intersect.Enums;
+using System;
+using Intersect.Server.Database.GameData;
+using Intersect.Server.Entities;
+
+namespace Intersect.Server.Core.Commands
+{
+    class SetVarCommand : ServerCommand
+    {
+        public SetVarCommand() : base(
+            Strings.Commands.SetVar,
+            new VariableArgument<string>(Strings.Commands.Arguments.VarGUID, RequiredIfNotHelp, true),
+            new VariableArgument<string>(Strings.Commands.Arguments.VarValue, RequiredIfNotHelp, true)
+        )
+        {
+        }
+
+        private VariableArgument<string> ServerVarID => FindArgument<VariableArgument<string>>(0);
+        private VariableArgument<string> ServerVarValue => FindArgument<VariableArgument<string>>(1);
+
+        protected override void HandleValue(ServerContext context, ParserResult result)
+        {
+            var serverVar = result.Find(ServerVarID);
+            var varVal = result.Find(ServerVarValue);
+
+            if (!string.IsNullOrEmpty(serverVar) && !string.IsNullOrEmpty(varVal))
```

Comment:
Invert the if (will be an or afterwards) and print not found and then return.

Reply:
Done